sys-apps/dstat/dstat-0.7.4-r2.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_{6,7}} )
+
+inherit python-r1
+
+DESCRIPTION="Versatile replacement for vmstat, iostat and ifstat"
+HOMEPAGE="http://dag.wieers.com/home-made/dstat/"
+SRC_URI="https://github.com/dagwieers/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm64 hppa ~ia64 ~mips ppc ppc64 sparc x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples wifi"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-python/six[${PYTHON_USEDEP}]
+	wifi? (
+		net-wireless/python-wifi
+	)"
+DEPEND="${RDEPEND}"
+
+PATCHES=( \
+	"${FILESDIR}/dstat-${PV}-skip-non-sandbox-tests.patch" \
+	"${FILESDIR}/fix-collections-deprecation-warning.patch" \
+	"${FILESDIR}/dstat-fix-shebang.patch" \
```

Comment:
Please also include version number in the patch.

Reply:
Thanks juippis:

1. version number is now in the patch
2. rebased

I left it as a revbump because the test failure fix is a runtime fix as well (the shebang was installed incorrectly).

Thanks again,

Paul